ui/app/src/context/DarkMode.tsx

```diff
@@ -46,12 +51,16 @@ export function DarkModeContextProvider(props: { children: React.ReactNode }) {
   );
 
   const theme = useMemo(() => getTheme(isDarkModeEnabled ? 'dark' : 'light'), [isDarkModeEnabled]);
+  const chartsTheme: PersesChartsTheme = useMemo(() => {
+    return generateChartsTheme('perses', theme, ECHARTS_THEME_OVERRIDES);
+  }, [theme]);
 
   return (
     <ThemeProvider theme={theme}>
       <CssBaseline />
-
-      <DarkModeContext.Provider value={darkModeContext}>{props.children}</DarkModeContext.Provider>
+      <ChartsThemeProvider themeName="perses" chartsTheme={chartsTheme}>
```

Comment:
I moved `ChartsThemeProvider` up to the app-level, rather than keeping it in a view since it's basically a global provider.

Reply:
This still works with using the MUI theme as a param to generate the ECharts theme right?

Reply:
ohh nvm I see now, this makes sense and there's a unit test for this too